src/test/mod.rs

```diff
@@ -339,8 +348,11 @@ fn test_hover_after_src_line_change() {
     expect_messages(
         results.clone(),
         &[
-            ExpectedMessage::new(Some(13))
-                .expect_contains(r#"[{"language":"rust","value":"&str"}]"#),
+            ExpectedMessage::new(None).expect_contains("contents"),
+            ExpectedMessage::new(None).expect_contains("progress").expect_contains(r#"title":"Diagnostics""#),
+            // XXX why don't we get this?
```

Comment:
The rebase is done. But I couldn't figure out why this test changed. I might have broken something.

Reply:
Is there another progress message which appears before the hover message? Could you post the actual and expected output as produced by the test please?